config/onprem-iso-fcc.yaml

```diff
@@ -17,7 +30,7 @@ systemd:
             [Service]
             Type=forking
             Restart=no
-            ExecStart=podman pod create --name assisted-installer -p 5432,8000,8090,8080
+            ExecStart=podman pod create --name assisted-installer -p 5432,8000,8090,8443
```

Comment:
Maybe this is the right time to drop the rest of those ports. WDYT? Nothing outside of the pod should need to connect to them, except in the case of troubleshooting.

Reply:
+1

Reply:
Done!